### PR TITLE
GiftCard API Transactions endpoints updated

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [X] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [ ] No, but I am going to.

PR created to update field descriptions in GiftCard API endpoints (EDU-10227 and EDU-10198).

### CODE REVIEW

**Review by julia-rabello — CHANGES_REQUESTED** [review]

File: VTEX - Giftcard API.json [hunk 1]
```diff
@@ -147,33 +147,70 @@
...
+                                        "balance",
+                                        "emissionDate",
+                                        "expiringDate",
+                                        "transactions"
+                                    ],
+                                    "type": "object",
+                                    "properties": {
+                                        "id": {
+                                            "type": "string",
+                                            "description": "Gift card `id`."
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:179]: Modificado para Gift card identification.

File: VTEX - Giftcard API.json [hunk 2]
```diff
@@ -147,33 +147,70 @@
...
+                                        },
+                                        "redemptionToken": {
+                                            "type": "string",
+                                            "description": "Gift card redemption token."
+                                        },
+                                        "redemptionCode": {
+                                            "type": "string",
+                                            "description": "Gift card redemption token."
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:187]: Aqui foi um erro de duplicação mesmo. O correto para a segunda descrição é Gift card redemption code.

File: VTEX - Giftcard API.json [hunk 3]
```diff
@@ -147,33 +147,70 @@
...
+                                        },
+                                        "balance": {
+                                            "type": "integer",
+                                            "description": "Giftcard current balance. For Gift Cards newly created, the balance will be 0.0.",
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:191]: Fixed.

File: VTEX - Giftcard API.json [hunk 4]
```diff
@@ -1945,23 +1992,71 @@
...
                     },
                     "description": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "Description information."
                     },
                     "date": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "Date information."
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:2004]: Vou utilizar o padrão que vi em outros endpoints -> "Date of the transaction (YYYY-MM-DDThh:mm:ss.fff (ISO 8601 format)."

File: VTEX - Giftcard API.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:191]: Neste caso 0.0 mesmo.

File: VTEX - Giftcard API.json [hunk 5]
```diff
@@ -1945,23 +1992,71 @@
...
                 "properties": {
                     "value": {
-                        "type": "number"
+                        "type": "number",
+                        "description": "Value information."
                     },
                     "description": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "Description information."
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:2000]: Transaction description.

File: VTEX - Giftcard API.json [hunk 6]
```diff
@@ -2029,10 +2132,12 @@
...
+                        "type": "number",
+                        "description": "Value information."
                     },
                     "requestId": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "VTEX request identifier."
                     }
                 },
                 "default": {
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:2143]: Fixed.

File: VTEX - Giftcard API.json [hunk 7]
```diff
@@ -2012,10 +2113,12 @@
...
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:2124]: Fixed.

File: VTEX - Giftcard API.json [hunk 8]
```diff
@@ -1945,23 +1992,71 @@
                 "type": "object",
                 "properties": {
                     "value": {
-                        "type": "number"
+                        "type": "number",
+                        "description": "Value information."
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:1996]: Transaction value.

File: VTEX - Giftcard API.json [hunk 9]
```diff
@@ -1945,23 +1992,71 @@
...
+                        "properties": {
+                            "href": {
+                                "type": "string",
+                                "description": "Giftcard resource URL."
+                            }
+                        }
+                    },
+                    "operation": {
+                        "type": "string",
+                        "description": "Operation information."
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:2042]: Operation information (`credit` or `debit`).

File: VTEX - Giftcard API.json [hunk 10]
```diff
@@ -2029,10 +2132,12 @@
                 "type": "object",
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:2132]: Boa idéia. Deletei o `CancelGiftCardTransactionRequest` e modifiquei as chamadas do `SettleGiftCardTransactionRequest` para `GiftCardTransactionRequest`.

File: VTEX - Giftcard API.json [hunk 11]
```diff
@@ -176,19 +176,19 @@
...
                                             "type": "string",
                                             "description": "Gift card redemption token."
                                         },
                                         "redemptionCode": {
                                             "type": "string",
-                                            "description": "Gift card redemption token."
+                                            "description": "Gift card redemption code."
                                         },
                                         "balance": {
                                             "type": "integer",
```
Comment (julia-rabello) [inline VTEX - Giftcard API.json:190]:
```suggestion
                                            "type": "number",
```

File: VTEX - Giftcard API.json [hunk 12]
```diff
@@ -176,19 +176,19 @@
...
-                                            "description": "Giftcard current balance. For Gift Cards newly created, the balance will be 0.0.",
+                                            "description": "Giftcard current balance. For newly created giftcards, the balance will be 0.0.",
                                             "format": "int32"
```
Comment (julia-rabello) [inline VTEX - Giftcard API.json]:
```suggestion
                                            "description": "Giftcard current balance. For newly created giftcards, the balance will be 0.0."
```

File: VTEX - Giftcard API.json [hunk 13]
```diff
@@ -2074,7 +2073,7 @@
                     },
                     "date": {
                         "type": "string",
-                        "description": "Date information."
+                        "description": "Date of the transaction (YYYY-MM-DDThh:mm:ss.fff (ISO 8601 format)."
```
Comment (julia-rabello) [inline VTEX - Giftcard API.json:2076]:
```suggestion
                        "description": "Date of the transaction in [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) format (`YYYY-MM-DDThh:mm:ss.fff`)."
```

File: VTEX - Giftcard API.json [hunk 14]
```diff
@@ -2098,7 +2097,7 @@
...
```
Comment (julia-rabello) [inline VTEX - Giftcard API.json:2100]:
```suggestion
                        "description": "Date of the transaction in [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) format (`YYYY-MM-DDThh:mm:ss.fff`)."
```

File: VTEX - Giftcard API.json [hunk 15]
```diff
@@ -1988,20 +1988,19 @@
...
                     },
                     "description": {
                         "type": "string",
-                        "description": "Description information."
+                        "description": "Transaction description."
                     },
                     "date": {
                         "type": "string",
-                        "description": "Date information."
+                        "description": "Date of the transaction (YYYY-MM-DDThh:mm:ss.fff (ISO 8601 format)."
```
Comment (julia-rabello) [inline VTEX - Giftcard API.json:2003]:
```suggestion
                        "description": "Date of the transaction in [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) format (`YYYY-MM-DDThh:mm:ss.fff`)."
```

**Review by julia-rabello — CHANGES_REQUESTED** [review]

File: VTEX - Giftcard API.json (hunk 11 above)
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:190]: Adjusted.

File: VTEX - Giftcard API.json (hunk 13 above)
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:2076]: Adjusted.

File: VTEX - Giftcard API.json (hunk 15 above)
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:2003]: Adjusted.

File: VTEX - Giftcard API.json (hunk 14 above)
Comment (ricardoaerobr) [reply VTEX - Giftcard API.json:2100]: Adjusted.

File: VTEX - Giftcard API.json [hunk 16]
```diff
@@ -187,7 +187,7 @@
                                             "description": "Gift card redemption code."
                                         },
                                         "balance": {
-                                            "type": "integer",
+                                            "type": "number",
                                             "description": "Giftcard current balance. For newly created giftcards, the balance will be 0.0.",
                                             "format": "int32"
```
Comment (julia-rabello) [inline VTEX - Giftcard API.json]:
```suggestion
                                            "description": "Giftcard current balance. For newly created giftcards, the balance will be 0.0."
```

**Review by julia-rabello — CHANGES_REQUESTED** [review]

File: VTEX - Giftcard API.json [hunk 17]
```diff
@@ -147,33 +147,70 @@
...
+                                            "description": "Gift card redemption token."
+                                        },
+                                        "redemptionCode": {
+                                            "type": "string",
+                                            "description": "Gift card redemption code."
+                                        },
+                                        "balance": {
+                                            "type": "number",
+                                            "description": "Giftcard current balance. For newly created giftcards, the balance will be 0.0.",
+                                            "format": "int32"
```
Comment (julia-rabello) [inline VTEX - Giftcard API.json:192]:
```suggestion
                                            "description": "Giftcard current balance. For newly created giftcards, the balance will be 0.0."
```

**Review by julia-rabello — CHANGES_REQUESTED** [review]

**Review by julia-rabello — DISMISSED** [review]

**Review by julia-rabello — APPROVED** [review]

**Pull request merged**